sources/il.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "url": "http://www.svivaaqm.net/",
+    "adapter": "envista_il",
+    "name": "Israel",
+    "country": "IL",
+    "city": "FIXME",
```

Comment:
we could use the regions the stations are in as the `regional approximates` discussed in the data format repo